components/IconLink.js

```diff
@@ -82,14 +107,15 @@ const IconLink = ({
   mobileOnly,
   style,
   title,
-  onClick
+  onClick,
+  stacked
 }) => {
   const Icon = ICONS[icon]
 
   return (
     <a
       {...styles.link}
-      {...(mobileOnly ? styles.mobileOnly : {})}
+      {...(getExtraStyles(mobileOnly, stacked))}
```

Comment:
It's just a small thing, but I think the conditional styles should me merged into the `styles` object (with `css` already applied), and the test should be performed inline.

    {...merge(
       stacked && styles.stacked, 
       mobileOnly && styles.mobileOnly
    )}

To my eyes this would improve readability.

Reply:
@garamond I agree with the readability aspect, but we tend to avoid merge() at render time for performance reasons.